health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/NewCampaignCreate/CampaignDetails.js

```
@@ -120,24 +120,56 @@ const CampaignDetails = () => {
+          <HeaderComponent className={"date-header"}>{campaignData?.campaignName}</HeaderComponent>
+          <div
+            className="hover"
+            onClick={() => {
+              history.push(`/${window.contextPath}/employee/campaign/create-campaign?key=2&editName=${true}&id=${campaignData?.id}`);
+            }}
+          >
+            <Edit />
+          </div>
+        </div>
+        <div style={{ display: "flex" }}>
+          <Tag label={campaignData?.campaignName} showIcon={false} className={"campaign-view-tag"} type={"warning"} stroke={true}></Tag>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)